src/modules/map/components/LocationPopupCopySlot.vue

```diff
@@ -1,48 +1,51 @@
 <template>
-    <!-- eslint-disable vue/no-v-html-->
     <button
+        id="copyButton"
         ref="button"
         class="btn btn-light btn-sm"
         type="button"
         @click="copyValue"
-        v-html="$t(buttonText)"
-    />
-    <!-- eslint-enable vue/no-v-html-->
+    >
+        <FontAwesomeIcon class="icon" :icon="['far', 'copy']" />
+    </button>
 </template>
 
 <script>
 import log from '@/utils/logging'
+import tippy from 'tippy.js'
+import 'tippy.js/dist/tippy.css' // optional for styling
 
 export default {
     props: {
         value: {
             type: String,
             default: '',
         },
-        resetDelay: {
-            type: Number,
-            default: 1000,
-        },
-    },
-    data() {
-        return {
-            buttonText: 'copy_cta',
-        }
     },
-    unmounted() {
-        clearTimeout(this.resetTimeout)
+    mounted() {
+        tippy('#copyButton', {
+            content: this.$i18n.t('copy_cta'),
+            arrow: true,
+            placement: 'right',
+            touch: 'hold',
+        })
+        tippy('#copyButton', {
+            content: this.$i18n.t('copy_done'),
+            arrow: true,
+            placement: 'right',
+            trigger: 'click',
+            onShow(instance) {
+                setTimeout(() => {
+                    instance.hide()
+                }, 1000)
+            },
+            allowHTML: true,
```

Comment:
not sure if this flag is required, as it is only text that is being shown

Reply:
This is required because of non breaking space character in the `copied!` translation.